docs/topics/proc_updating-booster-deployment-configuration.adoc

```diff
@@ -0,0 +1,72 @@
+[[proc_updating-booster-deployment-configuration]]
+= Updating the Deployment Configuration of a Booster
+
+The deployment configuration for a booster contains information related to deploying and running the booster in OpenShift, such as route information or readiness probe location. 
+The deployment configuration of a booster is stored in a set YAML files. 
+For boosters that use the Fabric8 Maven Plugin, the YAML files are located in the `src/main/fabric8/` folder. For {NodeJS} boosters using Nodeshift, the YAML files are located in the `.nodeshift` folder.
+
+.Prerequisite
+* An existing booster project.
+* The `oc` CLI client installed.
+
+.Procedure
+
+. Edit the an existing YAML file or create an additional YAML file with your configuration update.
+** For example, if your booster already has a YAML file with a `readinessProbe` configured, you could change `path` value to different available path to check for readiness:
++
+[source,yaml,options="nowrap",subs="attributes+"]
+----
+spec:
+  template:
+    spec:
+      containers:
+        readinessProbe:
+          httpGet:
+            path: /path/to/probe
+            port: 8080
+            scheme: HTTP
+...
+----
+** If a `readinessProbe` is not configured in an existing YAML file, you can also create a new YAML in the same directory with the `readinessProbe` configuration.
+
+. Deploy the updated version of your booster using Maven or `npm`.
```

Comment:
I wouldn't put npm in verbatim because Maven isn't. Not that it's wrong, it just looks better IMO.